src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -355,6 +356,7 @@ private void AddTelemetryProvider(IServiceCollection services)
         {
             var configuration = new TelemetryConfiguration();
             Configuration.GetSection("Telemetry").Bind(configuration);
+            services.AddTransient<IMeterFactory, DummyMeterFactory>();
```

Comment:
@fhibf this failed to resolve for me when running locally against cosmos, any ideas?